evalai/utils/requests.py

```diff
@@ -11,75 +10,58 @@
 
 
 def make_request(path, method, files=None, data=None):
+    if method in  ["PUT", "PATCH", "DELETE"]:
+        echo(
+            style(
+                "\nError: HTTP Method is not supported: {}\n".format(method),
+                bold=True,
+                fg="red",
+            )
+        )
+        sys.exit(1)
```

Comment:
Don't echo, throw error in this case

Reply:
Okay. `ValueError`?

Reply:
Why value error?

Reply:
I think that can be appropriate because the `method` parameter got an invalid value as an argument. 
What do you say?

Reply:
@vkartik97 It can more appropriately be called 'UnsupportedMethodError' or 'InvalidArgumentError' but since we don't have custom exception types, 'ValueError' can be used (as the InvalidArgumentError can be considered a form of ValueError).

Otherwise, I think just calling it an `Exception` would be okay and clear all confusion. 
We can also create custom exception types if it can be used in multiple cases. This can be the best thing to do but I'm not sure if it's feasible if it only has one use case. 
What is your opinion on this?